app/models/user.rb

```diff
@@ -2,10 +2,11 @@
 
 class User < ApplicationRecord
   has_secure_password
-
   has_many :rooms, dependent: :destroy
-
-  validates :name, presence: true
+  validates :name, presence: true # TODO: amir - Change into full_name or seperate first and last name.
```

Comment:
In terms of database normalization some may consider this as not normalized because it violates the 1st normal form:
`If a relation contains a composite or multi-valued attribute, it violates the first normal form, or the relation is in first normal form if it does not contain any composite or multi-valued attribute. A relation is in first normal form if every attribute in that relation is singled valued attribute. `
https://www.geeksforgeeks.org/first-normal-form-1nf/

1. The semantic name should be descriptive -> `full_name` rather than `name`.
2. Having two attributes one for the first name and the other for the last name may simplify validations and filtering (i.e. won't have to worry about whitespaces and its implications for length and representation).
3. For database administrators to better filter data and query it along with a better use of memory (see database normalization).
But this is not a big deal in our case at least it should be name `full_name` IMHO.

Reply:
So do I leave the todo comment there?

Reply:
Yeah

Reply:
Done,